desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java

```diff
@@ -273,13 +268,11 @@ private void addContent() {
                         Res.get(contract.getPaymentMethodId()));
         }
 
-        addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.makerFeeTxId"), offer.getOfferFeePaymentTxId());
-
         if (trade.getMakerDepositTx() != null)
-            addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.depositTransactionId"), // TODO (woodser): separate UI labels for deposit tx ids
+            addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.yourDepositTransactionId"),
```

Comment:
Should be `shared.makerDepositTransactionId` and `shared.takerDepositTransactionId` here.